app/experimenter/experiments/api/v5/types.py

```diff
@@ -405,6 +408,39 @@ def resolve_targeting_config_slug(self, info):
             return self.TargetingConfig(self.targeting_config_slug).value
         return self.targeting_config_slug
 
+    def resolve_targeting_configs(self, info):
```

Comment:
Drive by sick review 😷 

This looks great except for one thing:

This should be in the `getConfig` query, not `getExperimentBySlug`, since it's all targeting configs, not just the one specific to this experiment.

Reply:
The field is added both in `getConfig` and `getExperimentBySlug`, the idea is when we set the target config slug to the experiment then it will return that particular targeting config, else will return list of targeting config

Reply:
so should we restrict to `getConfig` or can we keep in both?

Reply:
Oh yeah you updated the `getConfig` query, so you don't need the second case (lines 428-442) here.  The NimbusExperimentType only needs to serialize the data for a single experiment.  And instead of looping and matching the slug you can just instantiate it with the value the same way you did on line 306.